package.json

```diff
@@ -104,6 +104,7 @@
     "@types/request": "^2.48.3",
     "adf-monaco-extension": "0.0.8",
     "adf-tslint-rules": "0.0.6",
+    "angular2-uuid": "^1.1.1",
```

Comment:
Homepage says "THIS PROJECT IS NOT MAINTAINED", can any other library be used? @eromano maybe we can move UUID generation helpers from e2e to the core?

Reply:
do you guys have a library or utility that generates random IDs? pretty simple to throw together if not.

Reply:
I don't think is needed anymore as the methods and the import where it is used

Reply:
I don't think is needed anymore as the methods and the import where it is used